### PR TITLE
Added Orientation Button and Script

### PR DESCRIPTION
**Pull request by Jmac92**
Switches between isometric and third-person camera views when the button is pressed. Working as intended

### CODE REVIEW

**Comment by Jmac92** [comment]
I also changed the map style to one of the map box defaults for now due to the custom one breaking using hd textures and looking awful every time I hit play

**Comment by LMDyer** [comment]
Could you provide some screenshots please? How badly was the custom map breaking things?

File: FinderKeeper/Assets/Scripts/CameraSwap.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraSwap : MonoBehaviour
+ {
+     public Camera IsometricCamera, ThirdPersonCamera;
+     public bool swapFlag = false;
```
Comment (LMDyer) [inline FinderKeeper/Assets/Scripts/CameraSwap.cs]: Just being picky, but it's not clear what value "swapFlag" implies from the name. Maybe a name like "isIsoCamera" would be clearer?

**Comment by Jmac92** [comment]
screen shot 2018-08-09 at 12 25 11 pm
Isometric view

**Comment by Jmac92** [comment]
screen shot 2018-08-09 at 12 25 23 pm
Third person view

File: FinderKeeper/Assets/Scripts/CameraSwap.cs (hunk 1 above)
Comment (Jmac92) [reply FinderKeeper/Assets/Scripts/CameraSwap.cs]: Half the time the custom map just shows up as the black loading tiles, the other half it is extremely fuzzy. So I just changed the drop down from custom to a default style for testing purposes. Will need to investigate further.

Comment (Jmac92) [reply FinderKeeper/Assets/Scripts/CameraSwap.cs]: That's fair. I'll think of a clearer name for it

**Pull request merged**